ada/language/ast.py

```diff
@@ -901,6 +901,24 @@ def library_item_pragmas():
             No(T.Pragma.list.entity)
         )
 
+    @langkit_property(return_type=T.AdaNode.entity, public=True)
+    def get_attribute(name=SymbolType):
+        """
+        Return the attribute with name ``name`` associated to this entity.
+
+        Attribute is the term chosen to unify data that can be set by the user
+        on an Ada entity via three different mechanisms, sometimes
+        interchangeably: Pragmas, aspects and representation clauses.
+
+        This will return the syntactic node corresponding to attribute
```

Comment:
Does this sentense misses a word? (“to the attribute”) I’m having trouble parsing it :)